src/main/java/frc/robot/Feedback/LED/Limelight.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.Feedback.LED;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class Limelight extends CommandBase {
+  /** Creates a new Limelight. */
+  private TowerSubsystem m_tower;
+  public Limelight(TowerSubsystem m_tower) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    public static NetworkTableInstance table = NetworkTableInstance.getDefault();
+    this.m_tower = m_tower;
+    addRequirements(m_tower);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if (!m_tower.m_entryBeamBreak) {
```

Comment:
My suggestion would be to make this command not specific to the Tower, so if other subsystems want to flash the Limelight LEDs they can do so using this command. So to do this, you can put the logic that checks the beam break state, and then schedules this command in [TowerSubsytem](https://github.com/WHS-FRC-3467/Skip-5.12/blob/master/src/main/java/frc/robot/subsystems/Tower/TowerSubsystem.java#L64) using the WPILib command [`schedule()`](https://docs.wpilib.org/en/stable/docs/software/commandbased/command-scheduler.html) method.